tests/test_auxtel_track_target.py

```diff
@@ -46,6 +47,16 @@ async def basic_make_script(self, index):
 
         return (self.script,)
 
+    async def test_configure_with_slew_planet(self):
+        async with self.make_script():
+            self.script._atcs = unittest.mock.AsyncMock()
+            self.script._atcs.start_task = utils.make_done_future()
+
+            # Script can be configured with slew_planet
+            await self.configure_script(
+                slew_planet=dict(planet_name="PLUTO", rot_sky=0.0)
```

Comment:
You removed `rot_sky` right?

Reply:
I realized that while running the unit test. All unit tests passing now.